fusestore/store/src/api/rpc/flight_service_test.rs

```diff
@@ -583,3 +587,153 @@ async fn test_flight_generic_kv() -> anyhow::Result<()> {
 
     Ok(())
 }
+
+#[tokio::test(flavor = "multi_thread", worker_threads = 1)]
+async fn test_flight_get_database_meta_empty_db() -> anyhow::Result<()> {
+    common_tracing::init_default_tracing();
+    let (_tc, addr) = crate::tests::start_store_server().await?;
+    let mut client = StoreClient::try_create(addr.as_str(), "root", "xxx").await?;
+
+    // Empty Database
+    let res = client.get_database_meta(None).await?;
+    assert_eq!(None, res);
```

Comment:
Without any databases, a result `(0, vec![])` may be more convenient to use, i.e., one less `match` on an `Option<_>`.

Reply:
:D  A version of `get_database_meta` did return `(0, vec![])` when there is nothing to sync according to the params which client provides.  
But I changed it to the current signature (and return-type),  since:
-  `(0, res)` where `res` is not empty might be confusing to the caller site 
     The caller may need extra treatments, depends on the caller be defensive or else.
-   `None` is slightly efficient while being serialized

Reply:
All right